src/net_processing.cpp

```diff
@@ -1129,6 +1132,12 @@ void PeerManagerImpl::RemoveBlockRequest(const uint256& hash)
     }
 
     auto [node_id, list_it] = it->second;
+
+    if (from_peer && node_id != *from_peer) {
+        // Block was requested by another peer
```

Comment:
nit: requested **from** another peer

Reply:
Oops -- will fix in a followup. Thanks for catching.

Reply:
"by another peer" reads fine to me?

Reply:
ah right, suggested text is also ambiguous....

"Block was requested by us from another peer" ?